po/ChangeLog

```diff
@@ -1,3 +1,6 @@
+2013-06-08: v0.1: Added Dutch translation
```

Comment:
It doesn't make any sense either.

Be careful when you commit! You can use `git add -p` and `git commit -v` to review your changes before committing.

Reply:
forgive me I accidentally deleted it. well i will correction

Reply:
i was fix it

Reply:
forgive by accidentally deleting it

Reply:
maafkan saya

Reply:
I'm sorry, I'm careless